src/js/pensions/config/form.js

```diff
@@ -176,6 +177,66 @@ const formConfig = {
               nationalGuard: _.set('properties.address', address.schema(fullSchemaPensions), nationalGuard)
             }
           }
+        },
+        powAndSeverance: {
+          path: 'military/pow-severance',
+          title: 'POW Status & Severance Pay',
+          uiSchema: {
+            'ui:title': 'POW Status & Severance Pay',
+            'ui:order': ['powStatus', 'powDateRange', 'severancePay', 'payType', 'payAmount'],
+            powStatus: {
+              'ui:title': 'Have you ever been a POW?',
+              'ui:widget': 'yesNo'
+            },
+            powDateRange: _.set('ui:options.expandUnder', 'powStatus', dateRangeUI(
+                  'Start of confinement',
+                  'End of confinement',
+                  'Confinement start date must be before end date'
+            )),
+            severancePay: {
+              'ui:title': 'Have you received any type of severance or separation pay?',
+              'ui:widget': 'yesNo'
+            },
+            payType: {
+              'ui:title': 'Pay type',
+              'ui:widget': 'RadioWidget',
+              'ui:options': {
+                expandUnder: 'severancePay'
+              }
+            },
+            payAmount: {
+              'ui:title': 'Amount',
+              'ui:options': {
+                expandUnder: 'severancePay'
+              }
+            }
+          },
+          schema: {
+            type: 'object',
+            required: ['powStatus', 'severancePay'],
+            properties: {
+              powStatus: {
+                type: 'boolean'
+              },
+              powDateRange,
+              severancePay: {
```

Comment:
It doesn't look like this matches the backend schema. I see a `severancePay` object that contains amount and type, so we should follow that unless there's a reason not to. For this yes/no question you can do something like `view:receivedSeverancePay`.